CHANGES.rst

```diff
@@ -36,6 +36,9 @@ Release history
 - Added support for ``nengo.transforms.ConvolutionTranspose``. (`#300`_)
 - ``Block`` now has a ``.discretize_info`` attribute that stores parameters used
   for discretizing that block. (`#309`_)
+- ``Model`` now has a ``connection_decode_neurons`` attribute that maps ``Connection``
```

Comment:
Decode neurons are a bit of a sticking point for users coming to NengoLoihi from Nengo, since they don't exist at all in Nengo. Long term, I definitely think we need a better solution that can be accessed through Simulator. (When we started on NengoLoihi, I even thought that having a function that takes in a `nengo.Network` and returns a Loihi-compatible `Network` would be useful, since that could even be opened in e.g. NengoGUI to visualize, and things like DecodeNeurons could be probed.)

This is certainly a stop-gap solution, mostly for our internal use. At the same time, I'm not so worried about giving an impression that all Model changes are documented by mentioning this one. So I would probably keep it in, but I'm also fine to take it out if you prefer.

At some point, I'd like to put together a "tips and tricks"-type notebook that shows how to do some of the things we often get asked about on the forum. Inspecting decode neurons would be one, also measuring power usage.

Reply:
I feel like we have a lot of this stuff in the backlog somewhere... in any case, since the entry is there might as well keep it in.